src/components/TimestampForm/timestampform.module.css

```diff
@@ -0,0 +1,146 @@
+/* TimestampForm styles */
+
+.card {
+    background: #FFFFFF 0% 0% no-repeat padding-box;
+    box-shadow: 0px 4px 16px #48566E36;
+    opacity: 1;
+    width: 100%;
+    position: relative;
+    margin-top: 50px;
+}
+
+.content {
+    display: flex;
+    flex-direction: column;
+    margin-right: 30px;
+    margin-left: 30px;
+}
+
+.tabs {
+    display: flex;
+    z-index: 999;
+    width: 240px;
+    position: absolute;
```

Comment:
`.tabs` should only be absolute if they have a `position: relative` parent. Right now, its parent is the `.pageMain` which is a bad practice. I suggest that you wrap tabs in a relative positioned div.